exercise_01/transformStringToCollection.js

```diff
@@ -1,3 +1,4 @@
+import validateString from "../helpers/validateString.js";
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!
```suggestion
import validateString from "../helpers/validateString.js";

```